src/efmt/format.rs

```diff
@@ -228,6 +230,12 @@ impl Format {
                 let sub_str = &s[prev_idx..end_idx];
 
                 match prev_token {
+                    Token::YearShort => {
+                        decomposed[0] = sub_str
+                            .parse::<i32>()
+                            .map_err(|_| Errors::ParseError(ParsingErrors::ValueError))?
+                            + 2000;
```

Comment:
@ChristopherRabotin shouldn't `sub_str` be trimmed() here, so `.parse()` never fails ? 
In other terms, may whitespaces exist prior next separator

Reply:
Ah, I didn't think of this. I forget how it works. Don't we parse token by token and therefore it isn't possible to have a space? Do the other cases have a sub str? I'm afraid I'm busy this weekend and can't look into this.